tasks/tasks.go

```diff
@@ -38,12 +40,37 @@ func LoadStorageEngine(ctx context.Context, config *tsbridge.Config) (storage.Ma
 	}
 }
 
-// Sync updates all configured metrics.
-func Sync(ctx context.Context, config *tsbridge.Config, metrics *tsbridge.Metrics, store storage.Manager) error {
-	metricCfg, err := tsbridge.NewMetricConfig(ctx, config, store)
+// Checks if the metric config file has been updated, and updates the cached file stats if it has.
+func metricConfigFileUpdated(ctx context.Context, config *tsbridge.Config, metricCfgFs *os.FileInfo) (bool, error) {
```

Comment:
Is the necessary to pass the full config struct, when all you're reading is the filename?

Reply:
Changed this to pass only the filename :)